hub/examples/image_retraining/label_image.py

```diff
@@ -1,4 +1,4 @@
-import tensorflow as tf, sys
+import tensorflow.compat.v1 as tf,sys
```

Comment:
@kannandreams Please don't change it here, change it in the call only like `tf.compat.v1`

Reply:
oops ! my bad. change the code as per the comment.